### ISSUE TITLE
burned litter to atm and bgc?

### ISSUE THREAD

**Issue by rgknox** [NGEET/fates#485]
With the soon to be added nutrient cycle in FATES, this might be a good time to start scoping out the fate (haha!) of nutrients in burning plant matter.  As far as I can tell, everything that burns in FATES now, is assumed to be sent to the atmosphere.  Being a carbon only model, this doesn't seem like a huge deal.

My understanding is that newly burned areas with lots of ash should be pretty nutrient rich, and ripe for opportunists right?  

I think the software side of things depends on how and if the BGC models handles burned organic matter or if we should maintain some ash residence in fates.  I'm guessing that the BGC models have some highly mobile nutrient pools that we can send things to.   

Should we maintain an ash pool, or should we send mass fluxes in the ash straight to the BGC model for it to handle?

@jkshuman

**Comment by ckoven** [comment]
@rgknox Yes, I think we'd want to have a combustion completeness term that has dimensions of organ type (leaves burn more completely than wood) and nutrient type (carbon burns more completely than phosphorus), and then route the remainder straight to the BGC code?

**Comment by jkshuman** [comment]
@rgknox I like the idea of maintaining an ash pool. I still need to dig into this, but, as @ckoven, indicated there are differences in the amount of ash generated dependent on fuel and nutrient type. These will vary regionally and with fire behavior. I would need to look in the literature to get an idea of the relationship between the amount of ash left behind and fire characteristics (fire intensity, residence time, etc). This is a figure from Hurst et al 1994 where they measured emissions and combustion completeness for tropical savanna fires in Australia. There is a strong amount of partitioning, and large differences for fuel C and fuel N, with C burning more completely than N. @rgknox  Are you thinking we would use available nutrients in ash for regeneration? 

![hurst_etal_1994_fig4_fuel_c_n_partitioning](https://user-images.githubusercontent.com/22102918/53373276-38ffe600-3912-11e9-9290-82d786b377cd.png)

**Comment by rgknox** [comment]
@jkshuman, insightful figure.  Yeah I'm thinking that it would be good to capture the patch-level effects of post-fire nutrient concentrations on regeneration.  The trick, as I see it, is that any nutrient competition model the plants will partake in will be competing with below-ground microbes, which is resolved at the site level.

**Comment by lmkueppers** [comment]
Considering the fate of burned material as we develop the nutrient model is important. But it seems we should use a consistent approach for what is handled by the HLM BGC and what is handled by FATES or else there could be weird duplications. If detritus is now handled by the HLM BGC scheme, passing C and nutrients to that scheme seems best to me. So long as the new/envisioned land-use scheme can resolve the patch vs site differentiation of post-fire soil column needed to then reflect the post-fire BGC regime... Post-fire soil moisture regime also needs to be differentiated at the patch scale to get the post-fire regeneration right, so associating soil columns to patches has other benefits beyond the nutrients.
